src/mon/FSCommands.cc

```diff
@@ -107,7 +107,7 @@ class FailHandler : public FileSystemCommandHandler
 
   int handle(
       Monitor* mon,
-      FSMap& fsmap,
+      FSMap &fsmap,
```

Comment:
This is formatted this way because the reference `&` is part of the type. It helps to clearly separate the type from the variable name. This was a style recommended by Kefu.

Let's undo this change any others like it.

Reply:
Ok. Thank you for the explanation.